weave-js/src/components/PagePanelComponents/Home/Browse3/filters/FilterPanel.tsx

```diff
@@ -20,19 +21,17 @@ type FilterPanelProps = {
 };
 
 export const FilterPanel = (props: FilterPanelProps) => {
+  const {width: windowWidth} = useWindowSize();
+  // 400 = 60(sidebar) + 50 * 3(refresh, export, delete buttons) + 100 * 2 (compare + filter text)
+  let maxFilterBarWidth = windowWidth - 400;
+  if (!props.isEvaluateTable) {
+    // For op filter on traces table
+    maxFilterBarWidth -= 300;
```

Comment:
Something is wrong with the available width calculation on the evaluations page, the filters are not hiding when they should, though this appears broken in prod too.